src/pages/verify/integrations/firebase.mdx

```diff
@@ -0,0 +1,57 @@
+---
+product: verify
+category: Integrations
+sort: 3
+title: Firebase Authentication with Identity Platform
+subtitle: This tutorial demonstrates how to integrate Criipto Verify with Firebase Authentication with Identity Platform.
```

Comment:
Curious if we could get our eID keywords in here in a good way

Reply:
"This tutorial demonstrates how to integrate MitID, BankID and more eIDs with Firebase using Criipto Verify" perhaps?

Reply:
We do that for some of our guides, and for others we don't - Probably just need to pick one way of writing it.